src/js/utils/dom.js

```diff
@@ -49,17 +49,17 @@ export function htmlToParentElement(html) {
 }
 
 export function sanitizeScriptNodes(element) {
-    const scripts = element.querySelectorAll('script');
-    for (let i = 0; i < scripts.length; i++) {
-        const script = scripts[i];
-        script.parentNode.removeChild(script);
+    const nodes = element.querySelectorAll('script,object,iframe');
+    for (let i = nodes.length; i--;) {
+        const node = nodes[i];
+        node.parentNode.removeChild(node);
     }
     return element;
 }
 
 export function sanitizeElementAttributes(element) {
     const attributes = element.attributes;
-    for (let i = 0; i < attributes.length; i++) {
+    for (let i = attributes.length; i--;) {
```

Comment:
This is an important fix. `attributes.length` is decremented as attributes are removed, so we have to start at the end and cycle through in reverse to remove all matches.